openapi-raw.yaml

```diff
@@ -2099,216 +2099,6 @@ paths:
         seo:
           title: '_t__SignatureRequestCreateEmbeddedWithTemplate::SEO::TITLE'
           description: '_t__SignatureRequestCreateEmbeddedWithTemplate::SEO::DESCRIPTION'
-  '/signature_request/files/{signature_request_id}?get_url=1&file_type=pdf':
```

Comment:
This is stripping out a lot of existing work on the OAS. Could you merge in a fresh pull from the `oas-release` branch then re-build and re-push?

Reply:
Disregard. I'll clarify in standup.